### PR TITLE
1448 creation abstract toast class

### PR DESCRIPTION
**Pull request by NataliaTikhomirova**
https://github.com/jdi-testing/jdi-light/issues/1448

### CODE REVIEW

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/composite/Toast.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4622366&bid=15500183)

File: jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/composite/Toast.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    }
+
+    public String getMessage() {
+        return message.getText();
+    }
+
+    public void close() {
+        linked(".toast-header button", "Close").click();
+    }
+}
```
Comment (OBraunEPAM) [inline jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/composite/Toast.java]: Please add new line at the end of the file.

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/composite/Toast.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4622366&bid=15500183)

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/composite/Toast.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4622366&bid=15500183)

**Pull request merged**